src/extendedstorageobserver.h

```diff
@@ -57,28 +58,32 @@ class MKCAL_EXPORT ExtendedStorageObserver //krazy:exclude=dpointer
        is being observed.
        @param info uids inserted/updated/deleted, modified file etc.
     */
-    virtual void storageModified(ExtendedStorage *storage, const QString &info) = 0;
+    virtual void storageModified(ExtendedStorage *storage, const QString &info);
 
     /**
-       Notify the Observer that a Storage is executing an action.
-       This callback is called typically for example every time
-       an incidence has been loaded.
+       Notify the Observer that a Storage has finished an action.
 
        @param storage is a pointer to the ExtendedStorage object that
        is being observed.
+       @param error true if action was unsuccessful; false otherwise
        @param info textual information
     */
-    virtual void storageProgress(ExtendedStorage *storage, const QString &info) = 0;
+    virtual void storageFinished(ExtendedStorage *storage, bool error, const QString &info);
```

Comment:
Side-note: this is also slightly strange api when it's either load or save operation, no indication which.

Reply:
I can imagine the point of using this signal, even with an argument saying what is finished. But I was a bit reluctant to remove it completely. Maybe I'm missing something…